docs/rest.md

```diff
@@ -247,6 +247,14 @@ Which channel to search in.
    derived by Geo IP lookup.
 
 ## /v2/snaps
+
+### Parameters:
+
+#### `select`
+
+Filter from the given selection. Currently only limiting to refreshable
+snaps is supported via the `refresh` key.
```

Comment:
I think I agree that this belongs more to find at the REST API level if we want to maintain that /snaps doesn't need to talk remotely, otoh I don't like snap find --refresh that much as command line way to get to it

so snap refresh --list would seem again better